superset/connectors/sqla/models.py

```diff
@@ -47,7 +47,8 @@ def query(self, query_obj):
         error_message = None
         qry = db.session.query(Annotation)
         qry = qry.filter(Annotation.layer_id == query_obj['filter'][0]['val'])
-        qry = qry.filter(Annotation.start_dttm >= query_obj['from_dttm'])
+        if query_obj['from_dttm']:
+            qry = qry.filter(Annotation.start_dttm >= query_obj['from_dttm'])
```

Comment:
oh we need to same for `to_dttm` I think

Reply:
`to_dttm` is set to the current date in the query object. I can add the guard though 👍